GiniVision/Classes/Core/GiniVisionUtils.swift

```diff
@@ -134,21 +134,29 @@ internal class Constraints {
         constraint.isActive = true
     }
     
+    class func clip(view: UIView, toSuperView superview: UIView) {
```

Comment:
It's not "Clipping", is it. It's more like pinning, or filling the parent. IDK

Reply:
I doubted a lot choosing this name, and I decided to go with this because of the `clipToBounds` property. Maybe it is better the `pin` option.